starter-code/pizza.js

```diff
@@ -1 +1,74 @@
+
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function () {
+    //(iteration5)
+    var price = 13;
+    updatePizzaPrice();
+    //iteration1
+    $('.controls .btn-pepperonni').on('click', function (e) {
+        $('.pep').toggle();
+        $('.price li:nth-child(1)').toggle();
+    
+    });
+    $('.controls .btn-mushrooms').on('click', function (e) {
+        $('.mushroom').toggle();
+        $('.price li:nth-child(2)').toggle();
+        
+    });
+    $('.controls .btn-green-peppers').on('click', function (e) {
+        $('.green-pepper').toggle();
+        $('.price li:nth-child(3)').toggle();
+         
+    });
+
+    //iteration2
+    $('.crust-gluten-free').removeClass('crust-gluten-free');
+    $('.sauce-white').removeClass('sauce-white');
+
+    $('.controls .btn-crust').on('click', function (e) {
+        $('.crust').toggleClass('crust-gluten-free');
+    });
+
+    $('.controls .btn-sauce').on('click', function (e) {
+        $('.sauce').toggleClass('sauce-white');
+    });
+    //iteration3
+    $('.btn-pepperonni,.btn-mushrooms,.btn-green-peppers').addClass('active');
+    $('.btn-sauce,.btn-crust').removeClass('active');
+
+    $('.btn-pepperonni').on('click', function (e) {
```

Comment:
I think there is a lot of repetition, for the next time try to put the code below in one single function a pass params to change the diff values used